student.rb

```diff
@@ -1,9 +1,10 @@
 require './person'
+
 class Student < Person
-  attr_accessor :classroom
+  attr_reader :classroom
 
   def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
-    super(age: age, name: name, parent_permission: parent_permission)
+    super(name: name, age: age, parent_permission: parent_permission)
```

Comment:
- [ ] please like the previous reviewer noted there shoudl be method that serves as the 'belongs-to' side (a student belongs to a classroom)
eg 
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```